<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDeviceDialog</class>
 <widget class="QDialog" name="ConnectDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BLE蓝牙设备选择并连接</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QListWidget" name="list_Devices"/>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="list_Services"/>
     </item>
     <item row="1" column="2">
      <widget class="QListWidget" name="list_Characteristics"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设备：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_DeviceDiscoveryState">
         <property name="text">
          <string>就绪</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>服务：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_ServiceDiscoveryState">
         <property name="text">
          <string>就绪</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>特征：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_CharacteristicDiscoveryState">
         <property name="text">
          <string>就绪</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bt_Search">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bt_OK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
